README.md

```diff
@@ -48,8 +48,9 @@ Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 L
       - [LineageOS for microg](https://download.lineage.microg.org)
       - [BlissRoms](https://blissroms.org)
       - [PixelExperience](https://download.pixelexperience.org)
-    - TWRP Recovery:
+    - some recoveries:
```

Comment:
maybe change that to `Recoveries` :thinking:

Reply:
Yes, done. (was following same pattern as ROMs list)